.gitbook/contents/developer/features/price-feeds.md

```diff
@@ -215,3 +104,94 @@ Note: Unlike a feed contract where every data query remains on-chain, Hybrid Com
 _Goerli/Mainnet_: potentially everything, dependent on your source
 
 _Fee_: 0.01 BOBA per Hybrind Compute request
+
+<figure><img src="../../../assets/boba straw.png" alt=""><figcaption></figcaption></figure>
+
```

Comment:
I can completely remove this if we do not want to show this as an example. opinions?